modular_zubbers/code/datums/quirks/positive_quirks/venemous_bite/venomous_bite_quirk.dm

```diff
@@ -0,0 +1,86 @@
+/// Format: (reagent typepath -> list(amount to inject per bite, cooldown))
+GLOBAL_LIST_INIT(venomous_bite_choice_specs, list(
+	/datum/reagent/toxin/player_venom = list(5, 80 SECONDS),
+	/datum/reagent/toxin/venom = list(5, 180 SECONDS),
+	/datum/reagent/toxin/staminatoxin = list(5, 120 SECONDS),
+	/datum/reagent/toxin/carpotoxin = list(5, 60 SECONDS), // less powerful than the zootoxin
+	// medicine
+	/datum/reagent/medicine/epinephrine = list(3, 20 SECONDS),
+	/datum/reagent/medicine/omnizine = list(2, 30 SECONDS), // barely worth it due to the damage biting does
```

Comment:
```suggestion
	/datum/reagent/medicine/epinephrine = list(5, 20 SECONDS),
	/datum/reagent/medicine/omnizine = list(5, 30 SECONDS),
```

Reply:
I don't find the point to microdose chems. But the time can probally be modified to like a minute or something also if needed.

Reply:
Each bite does a flat 5 damage, the microdose and low cooldown is to limit the effectiveness of healing, since that 2u of omnizine can only just barely heal that - though Im not sure if thats what we really want

Reply:
i *will* raise it, but omnizine will have a larger cooldown

Reply:
yeah that's fine